configs/nextflow.hopper.config

```diff
@@ -155,7 +155,7 @@ profiles {
     params.panelhs = "sentieon driver -r ${params.genome_file} -t ${params.cpu_all} -i "
     params.panelhs2 = " --algo HsMetricAlgo --targets_list ${params.intervals} --baits_list ${params.intervals} hs_metrics.txt"
     params.bed_melt = "${params.refpath}/bed/MELT/Hg38.genes.bed"
-    params.mei_list = "${params.refpath}/annotation_dbs/MELT/mei_list.txt"
+    params.mei_list = "${params.refpath}/annotation_dbs/MELT/mei_list_withpriors"
```

Comment:
A file suffix is nice for ease of understanding. Not sure whether it is a tsv or txt, but could add a txt suffix as before?

Reply:
fine